### PR TITLE
Adding cask for Molsoft Browser

### PR DESCRIPTION
**Pull request by dmerrick**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Review by vitorgalvao — APPROVED** [review]

File: Casks/molsoftbrowser.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'molsoftbrowser' do
+  version '3.8-7d'
+  sha256 '4a831c7525b72f81138f99fc4f96a40a390f3809e7818f93ba43973291bdef28'
+
+  url "http://www.molsoft.com/getbrowser.cgi?product=icm-browser&act=give&mirror=molsoft&platform=mac64&ver=#{version}"
+  appcast 'https://www.molsoft.com/getbrowser.cgi'
+  name 'Molsoft Browser'
+  homepage 'https://www.molsoft.com/icm_browser.html'
```
Comment (vladimyr) [inline Casks/molsoftbrowser.rb:8]:
```suggestion
  url "https://molsoft.com/getbrowser.cgi?product=icm-browser&act=give&mirror=molsoft&platform=mac64&ver=#{version}"
  appcast 'https://molsoft.com/getbrowser.cgi?product=icm-browser&act=list'
  name 'Molsoft Browser'
  homepage 'https://molsoft.com/icm_browser.html'
```

File: Casks/molsoftbrowser.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  app 'MolsoftBrowser64.app'
+end
```
Comment (vladimyr) [inline Casks/molsoftbrowser.rb:11]:
```suggestion

  zap trash: [
               '~/.browser',
               '~/Library/Preferences/com.molsoft.plist',
               '~/Library/Preferences/icmbrowser64.plist',
             ]
end
```

File: Casks/molsoftbrowser.rb (hunk 1 above)
Comment (vladimyr) [reply Casks/molsoftbrowser.rb:8]: Invoking `/getbrowser.cgi` without query params won't return downloads list.

**Comment by dmerrick** [comment]
Thank you for the suggestions!

**Pull request merged**